zerver/tests/test_openapi.py

```diff
@@ -223,6 +223,7 @@ class OpenAPIArgumentsTest(ZulipTestCase):
         # test because of the variable name mismatch. So really, it's more of a buggy endpoint.
         '/user_groups/{group_id}',  # Equivalent of what's in urls.py
         '/user_groups/{user_group_id}',  # What's in the OpenAPI docs
+        '/export/realm/{export_id}',
```

Comment:
Move this earlier., it's not related to the `use_groups` block that has comments.

Reply:
In particular, should be next to /export/realm.

Reply:
Woops, silly mistake. Fixed.